db/schema.rb

```diff
@@ -467,6 +467,7 @@
     t.text "future_learn_organisation_memberships", default: [], array: true
     t.boolean "forgotten", default: false
     t.string "school_name"
+    t.string "auth0_id"
```

Comment:
Not sure why this schema change is in this ticket? Has this come from previous work on CRUP?

Reply:
Ah yeah, good spot, I will remove this